packages/test-utils/src/wrapper.js

```diff
@@ -421,7 +421,14 @@ export default class Wrapper implements BaseWrapper {
 
     Object.keys(data).forEach((key) => {
       // $FlowIgnore : Problem with possibly null this.vm
-      this.vm.$set(this.vm, [key], data[key])
+      if (typeof data[key] === typeof {} && data[key] !== null) {
+        Object.keys(data[key]).forEach((key2) => {
+          var newObj = Object.assign({}, data[key], data[key][key2])
+          this.vm.$set(this.vm, [key], newObj)
+        })
+      } else {
+        this.vm.$set(this.vm, [key], data[key])
```

Comment:
I think it is necessary to add the following line before this line.
`// $FlowIgnore : Problem with possibly null this.vm`